src/helper-modules/companies-house/wrap-companies-house-http-error-callback.ts

```diff
@@ -9,25 +9,12 @@ type CompaniesHouseHttpErrorCallback = (error: Error) => ObservableInput<never>;
 export const createWrapCompaniesHouseHttpGetErrorCallback =
   ({ messageForUnknownError, knownErrors }: { messageForUnknownError: string; knownErrors: KnownErrors }): CompaniesHouseHttpErrorCallback =>
   (error: Error) => {
-    if (error instanceof AxiosError && error.response && typeof error.response.data === 'object') {
-      const errorResponseData = error.response.data;
-      let errorMessage: string;
-
-      if (typeof errorResponseData.error === 'string') {
-        errorMessage = errorResponseData.error;
-      } else if (errorResponseData.errors && errorResponseData.errors[0] && typeof errorResponseData.errors[0].error === 'string') {
-        errorMessage = errorResponseData.errors[0].error;
-      }
-
-      if (errorMessage) {
-        const errorMessageInLowerCase = errorMessage.toLowerCase();
-
-        knownErrors.forEach(({ caseInsensitiveSubstringToFind, throwError }) => {
-          if (errorMessageInLowerCase.includes(caseInsensitiveSubstringToFind.toLowerCase())) {
-            return throwError(error);
-          }
-        });
-      }
+    if (error instanceof AxiosError && error?.response) {
```

Comment:
I really don't think we need this but we can keep it for now :)

Reply:
Discussed, keeping it as it is for consistency.